designer/server/src/main/scala/pl/touk/nussknacker/ui/process/processingtype/ProcessingTypeData.scala

```diff
@@ -71,11 +71,11 @@ object ProcessingTypeData {
       processingType: ProcessingType,
       componentDefinitionExtractionMode: ComponentDefinitionExtractionMode
   ) = {
-    val scenarioProperties = modelData.modelConfig
+    val scenarioProperties = deploymentScenarioPropertiesConfig ++ modelData.modelConfig
```

Comment:
There is some problem with key type in this map. I guess that it is a map of `PropertyName -> ScenarioPropertyConfig` instead of `ProcessingType -> ...`? The same problem is in `fragmentProperties`

Reply:
Switched ProcessingType to String (I noticed that we use String as property name and ParameterName as component parameter name). Left "todo" to fix it in next PR.